src/data/builder-tools.js

```diff
@@ -802,6 +802,14 @@ export const Showcases = [
     getstarted: "/docs/operate-a-stake-pool/frankenwallet",
     tags: ["operatortool"],
   },
+  {
+    title: "Bakrypt.io",
+    description: "Bakrypt offers backend tools and Cloud storage services for brands, companies, and creators to help them build their NFTs.",
+    preview: require("./builder-tools/bakrypt-io.png"),
```

Comment:
This image is cut off in the list of project thumbnails because of the great amount of vertical white space (if that blank space were transparent it would be automatically clipped and then matted white on display).  The image cells are over 2:1 aspect ratio, so if you can remove most of the white background it will display correctly.

Reply:
oh I see! I'll be updating the image later today. Thank you